src/app/handler/trades.js

```diff
@@ -870,8 +870,10 @@ exports.offerChanged = function (offer, oldState) {
         require('./crafting').keepMetalSupply();
 
         // Sort inventory
-        require('../crafting').sortInventory(3);
-
+        if (process.env.SKIP_SORTING_INVENTORY === 'false'){
```

Comment:
The bot should sort the inventory by default. So it should be:

```js
process.env.SKIP_SORTING_INVENTORY !== 'false'
```

Reply:
You mean !== true

Reply:
`!== 'true'`
Shouldn't it be 'true' since it's 'false'?

Reply:
Yes

Reply:
Yeah it should have been `!== 'true'` my bad